pkg/platform/model/buildplanner.go

```diff
@@ -29,8 +29,9 @@ import (
 )
 
 const (
-	pollInterval = 1 * time.Second
-	pollTimeout  = 30 * time.Second
+	pollInterval       = 1 * time.Second
+	pollTimeout        = 30 * time.Second
+	buildStatusTimeout = 30 * time.Minute
```

Comment:
Not sure about this timeout. Ideally, we would be using the buildlogstreamer.

Reply:
```suggestion
	buildStatusTimeout = 24 * time.Hour
```

Let's set this much higher. We don't really know yet what other potential use-cases there are. And technically the client shouldn't care if you run it for several hours, long as we can communicate the API timing out.